dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/utils/CollectionUtils.java

```diff
@@ -96,4 +97,109 @@ public static <T extends Object> List<Map<String, Object>> getListByExclusion(Li
         return instanceList;
     }
 
+    /**
+     * Helper class to easily access cardinality properties of two collections.
+     *
+     * @param <O> the element type
+     */
+    private static class CardinalityHelper<O> {
```

Comment:
We need to remove this, and directly use `org.apache.commons.collections4.CollectionUtils`

Reply:
done